CMakeLists.txt

```diff
@@ -172,65 +172,56 @@ project(GSI)
   if( USE_BASELIBS )
     find_package(Baselibs REQUIRED)
   else()
-    find_package( NetCDF REQUIRED)
-    if(NETCDF4)
-      if(CMAKE_MAJOR_VERSION GREATER 2)
-        find_package( ZLIB )
-      endif()
-      find_package( CURL )
-    endif()
+    find_package( NetCDF COMPONENTS C Fortran REQUIRED)
+    find_package( ZLIB )
+    find_package( CURL )
+#   set(NETCDF_C_LIBRARIES "${NETCDF_Fortran_LIBRARIES} ${NETCDF_C_LIBRARIES}")
   endif()
   if(( NOT HOST-WCOSS_D ) AND ( NOT HOST-WCOSS_C ) AND ( NOT HOST-Gaea ) ) 
-    message(" trying to find lapack, ${host}")
-    find_package( LAPACK )
+    find_package( LAPACK REQUIRED )
   endif()
 # build the WRF I/O libraries
   if(DEFINED ENV{GSIWRF_LIB})
     set(wrflib "$ENV{GSIWRF_LIB}" CACHE INTERNAL "WRFIO library" )
-  elseif(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/libsrc/wrflib)
-    add_subdirectory(libsrc/wrflib)
+  elseif(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/src/wrflib)
+    add_subdirectory(src/wrflib)
```

Comment:
"add_subdirectory(src/wrflib)":  Is it confirmed that wrflib will be moved into src/ instead of from the NCEPLIBS repository?

Reply:
I moved it there because all the other libsrc directories were removed and it seemed strange to keep only wrflib. I am not insistent that it stays in src, though. Ideally, it would also be removed and replaced with something from NCEPLIBS or just removed altogether, but I don't think either of those is currently an option.